lib/openshift/machine.rb

```diff
@@ -0,0 +1,14 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # represents Machine
+  class Machine < ProjectResource
+    RESOURCE = 'machines'
+
+    # returns the node name the machine linked to
+    def node(user: nil, cached: true, quiet: false)
```

Comment:
Better set name to `node_name` where we could have method `node` return a `Node` object.